site/en/blog/optimizing-lcp-using-signed-exchanges/index.md

```diff
@@ -0,0 +1,459 @@
+ 
+Instead, I find it helpful to "zoom in" on the potentially affected page loads, and frame it as, "SXGs affect X% of page views, improving their LCP by Y milliseconds at the 75th percentile."
+ 
+Currently, SXG prefetch only happens under certain conditions:
+  - Chromium browser (e.g. Chrome or Edge except on [iOS](https://chromium.googlesource.com/chromium/src.git/+/HEAD/docs/ios/user_agent.md)), version M87 or higher
+  - Android OS
+  - `Referer: google.com` or other [Google search domains](https://source.chromium.org/chromium/chromium/src/+/main:components/google/core/common/google_tld_list.h;l=13-42;drc=bea0acf022da996a8ff1dbdee378667a66c768e4). (Note that in Google Analytics, a referral tag applies to [all page views in the session](https://support.google.com/analytics/answer/6205762?hl=en#flowchart&zippy=%2Cin-this-article), whereas SXG prefetch only applies to the first page view, directly linked from Google Search.)
+ 
+{% Aside %}
+Google Search is planning to expand the contexts where SXG prefetch can occur, including  desktop OSes and more. This may increase the X% over time, and also means you may want to update your metrics to accommodate the change. When that happens, it should be announced on [webpackaging-announce](https://groups.google.com/g/webpackaging-announce).
+{% endAside %}
+ 
+Read on for how to measure "X% of page views" and "improving their LCP by Y milliseconds", but note that this is an area I'm still learning more about. If you have additional advice on how to capture SXG performance in Real User Monitoring (RUM), please let us know! [File a bug against developer.chrome.com](https://github.com/GoogleChrome/developer.chrome.com) with your suggested improvements.
+
+### Contemporary study
+
+When looking at production data, you can split page loads into SXG and non-SXG. Cloudflare ASX provides support for [HTML syntaxes](https://github.com/google/sxg-rs#sxg-only-behavior) that let you determine in JS whether the page load is SXG.
+
+However, it is essential to limit the set of page loads you look at, so that the non-SXG side matches the eligibility conditions for SXG, in order to avoid selection bias. Otherwise, all of the following would exist *only* in the set of non-SXG page loads, which may have innately different LCP:
+
+  - iOS devices — due to differences in hardware or network speed among the users who have these devices.
+  - older Chromium browsers — for the same reasons.
+  - desktop devices — for the same reasons or because the page layout causes a different "largest element" to be chosen.
+  - same-site navigations (visitors following links within the site) — because they can reuse cached subresources from the previous page load.
```

Comment:
```suggestion
- iOS devices — due to differences in hardware or network speed among the users who have these devices.
- older Chromium browsers — for the same reasons.
- desktop devices — for the same reasons or because the page layout causes a different "largest element" to be chosen.
- same-site navigations (visitors following links within the site) — because they can reuse cached subresources from the previous page load.
```